package.json

```diff
@@ -16,9 +16,8 @@
     "dev": "cozy-run-dev",
     "standalone": "cozy-run-standalone",
     "build": "webpack",
-    "format": "prettier --write 'src/**/*.{js,json}'",
     "precommit": "yarn lint",
-    "lint": "eslint .",
+    "lint": "eslint --fix .",
```

Comment:
Why should the linting task fix errors on this project?

Reply:
I don't know, this is a commit that comes from #13 that follows https://github.com/konnectors/cozy-konnector-template/commit/1625ebac240d8c92f47c1b5f119f15eded3a46c2 (poke @drazik )

Reply:
Since Prettier is now packaged with eslint-config-cozy-app, if you don't autofix, you will  surely have prettier errors on linting. Up to you to remove `--fix` and do the formatting with a dedicated script if you prefer.

Reply:
But if I have som unstaged changes with eslint errors, I can't commit staged changes. True story. So I have an edited `package.json` that I cannot commit and I need to stash on code rebase.